apps/wallet/src/shared/utils/index.ts

```diff
@@ -24,15 +23,7 @@ export function usePageView() {
         customRPC && apiEnv === 'customRPC' ? customRPC : apiEnv.toUpperCase();
 
     useEffect(() => {
-        // NOTE: This is a hack to work around hook timing issues with the Growthbook SDK.
-        // Issue: https://github.com/growthbook/growthbook/issues/915
-        setTimeout(() => {
```

Comment:
Just leaving a note here: This is technically still true, but we no longer have an initial render inconsistency, so this code is only relevant for network changes.